### PR TITLE
community-containers: Add NocoDB

### PR DESCRIPTION
**Pull request by docjyJ**
Do you prefer that I put the original image, or do it manually?

By doing it manually, I verify that an update is not breaking. And when Nextcloud Tables V2 is here, I will put the original image back.

### CODE REVIEW

**Comment by szaimen** [comment]
> Do you prefer that I put the original image, or do it manually?

Both are fine...

> By doing it manually, I verify that an update is not breaking. And when Nextcloud Tables V2 is here, I will put the original image back.

Sounds like a plan :)

**Comment by docjyJ** [comment]
Sorry for the copy and paste, I should have done a crtl+f ^^'.

**Comment by szaimen** [comment]
> Sorry for the copy and paste, I should have done a crtl+f ^^'.

all good!

File: community-containers/nocodb/readme.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+> - Field (Person, Tag, File...)
+> - See more here https://github.com/nextcloud/tables/issues/103 
+
+## NocoDb server
+This container bundles NocoDb without synchronization with Nextcloud.
+
+This is an alternative of **Airtable**.
+
+### Notes
+- You need to configure a reverse proxy in order to run this container since nocodb needs a dedicated (sub)domain! For that, you might have a look at https://github.com/nextcloud/all-in-one/tree/main/community-containers/caddy.
```
Comment (szaimen) [inline community-containers/nocodb/readme.md:17]: I guess the container needs an update then? Will you create a PR? Also, I guess we should update the readme of the container then...

Comment (szaimen) [reply community-containers/nocodb/readme.md:17]: done in https://github.com/szaimen/aio-caddy/pull/40

Comment (szaimen) [reply community-containers/nocodb/readme.md:17]: however still needs readme update for the caddy container

**Comment by docjyJ** [comment]
> Both are fine...

I prefer not to lose any data due to an update.

**Comment by szaimen** [comment]
> > Both are fine...
> 
> I prefer not to lose any data due to an update.

all right 👍

**Review by szaimen — APPROVED** [review]
LGTM, thanks a lot! :)

**Comment by szaimen** [comment]
This is now released with v9.1.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel

**Comment by docjyJ** [comment]
Tested in local with Podman.
Login and webUI work for me.

**Pull request merged**